src/libcore/str/mod.rs

```diff
@@ -1249,12 +1233,12 @@ generate_pattern_iterators! {
     forward:
         /// Created with the method [`split`].
         ///
-        /// [`split`]: ../../std/primitive.str.html#method.split
+        /// [`split`]: str::split
```

Comment:
The link in core is right, but in std is wrong.
For example, with my local build, in std, str::split point to <http://0.0.0.0:8000/home/lzutao/fork/rust/compiler/src/libcore/std/primitive.str.html#method.split>.
@jyn514 is investigating it.

Reply:
Fixed by #74077 .